packages/dds/tree/src/domains/json/jsonCursor.ts

```diff
@@ -93,7 +93,7 @@ const adapter: CursorAdapter<JsonCompatible> = {
 };
 
 /**
- * Used to read a Jsonable tree for testing and benchmarking.
+ * Used to read generic json compatible data for testing and benchmarking.
```

Comment:
I think this would be even more clear. Also if practical link the schema from the JSON domain (ex: link the union of possible JSON types which is the schema the root of the tree this views will conform with)
```suggestion
 * Used to read generic json compatible data as a tree in the JSON domain.
```

Reply:
I think the schema is linked in the @returns part of this doc comment, is this what you're asking for?

Reply:
No, it links ITreeCursorSynchronous, and JsonCompatible. Those are both redundant with thats in the type signature, and neither doesn anything to indicate what schema the returned tree has. For example if this returns a tree for an object, what would the tree schema for that object be? Would it be a Map node? If so, what is the identifier for that node? THose are things a caller for this needs to know to be able to use it correctly.

I was thinking something more like linking jsonRoot from src/domains/json/jsonDomainSchema.ts and/or the simple tree equlivelent

Reply:
If you look at the implementation of the adapters this uses, it its type callback it uses jsonArray.name for example. Users of this cursor must match that schema.